src/core-package/iframe/handle-iframe.ts

```diff
@@ -196,6 +199,27 @@ export async function handleLoadedImageSize({
             timeoutMs,
         });
 
+        // Get image dimension again since host size dimension has changed
```

Comment:
Sending the size again will impact performance of this element when there are potentially hundreds on a single page (which is it's expected use case). Not to worry though, I've found a simpler CSS-only solution!